fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidConstants.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.transfer.v1.fluid;
+
+/**
+ * Constants for fluid transfer. In general, 1 bucket = 81000 droplets.
+ */
+public final class FluidConstants {
+	public static final long BUCKET = 81000;
+	public static final long BOTTLE = 27000;
+	public static final long INGOT = 9000;
+	public static final long NUGGET = 1000;
+	public static final long DROPLET = 1;
+
```

Comment:
in my opinion we should define some more standards while we're here
```suggestion
	/**
	 * The number of droplets of a liquid resource needed to equal one block of the solid material for materials that require 9 'ingots' for one block.
	 * Eg. gold, diamonds, emeralds, iron, netherite, coal, slime
	 */
	public static final int BLOCK_9 = BUCKET;
	/**
	 * The number of droplets of a liquid resource needed to equal one block of the solid material for materials that require 4 'ingots' for one block.
	 * Eg. 1.17 copper, honey block, glowstone
	 */
	public static final int BLOCK_4 = INGOT * 4;
```

Reply:
Yes, more standards is good. And I don't want clashes.

Reply:
Wouldn't a honey block be `81000`? I'm not sure about glowstone, and copper is 9 now, so I'm not sure we should define this.

Reply:
doesn't honey take 4 honey? I forgot

Reply:
glowstone, quartz, and a few other things are still in 4s

Reply:
Honey is 4 bottles.

Reply:
well bottles right, so hm, either 
n * bottles = 4 * (1/3) = 4/3b = 108000dp, or honey bottles are different from regular bottles. Minecraft isn't internally consistent anyways so we'll just have to chose one

Reply:
Honey bottle can be different imo - I'd say 81000 droplets = 1 block is the standard, anything else should be derived from that.

Reply:
Added that convention to the javadoc of `FluidConstants`.